wdl2cwl/main.py

```diff
@@ -30,6 +30,96 @@ def get_ram_min(ram_min: str) -> int:
     return int(float(ram_min.strip()) * 1024)
 
 
+def get_command(
+    command: str,
+    unbound: List[str],
+    bound: List[str],
+    input_types: List[str],
+    input_names: List[str],
+) -> str:
+    """
+    Get command to be used in the bash script.
+
+    Need to add sep= and default= for expression placeholders
+    """
+    index = 0
+    new_command = ""
+    start_index = 0
+    end_index = 0
+
+    # continue till the end of the string
+    while index < len(command):
+
+        # if you have ~{
+        if command[index] == "~" and command[index + 1] == "{":
+            start_index = index + 2
+
+            # while loop to find index of }
+            while 1:
+                if command[index] == "}":
+                    end_index = index
+                    break
+                else:
+                    index += 1
+
+            # sub string containing everything inside ~{ and }
+            sub_str = command[start_index:end_index]
+
+            # if sub string has a concatenation
+            if "+" in sub_str:
+                split_str = sub_str.split("+")
+
+                for i in split_str:
+                    if i in input_names:
+                        index = input_names.index(i)
+                        data_type = input_types[index]  # get the data type of the input
+
+                        if data_type == "File":
+                            new_command += "$(inputs." + i + ".path)"
+                        else:
+                            new_command += "$(inputs." + i + ")"
+                    else:
+                        new_command += i.replace('"', "")
+
+            # to handle Expression Placeholder Options
+            elif ("true" and "false") in sub_str:
+                sub = sub_str.split("=")
+                true_value = (sub[1].split('"'))[1]
+                false_value = (sub[2].split('"'))[1]
+                input_name = (sub[2].split('"'))[2]
+                append_str = (
+                    "$(if ['${inputs."
+                    + input_name
+                    + "}' eq 'true' ] then echo "
+                    + true_value
+                    + " else echo "
+                    + false_value
+                    + " fi )"
+                )
+                new_command += append_str
```

Comment:
As shown at https://github.com/common-workflow-lab/wdl-cwl-translator/pull/17/files#diff-b5a65c2befbbd0b6453805272729b622f41b2be683f8258432e08f4370332265R90 & https://app.codecov.io/gh/common-workflow-lab/wdl-cwl-translator/compare/17/diff#D1L86 , this section is not covered by your tests. So you can remove it from this PR, or add a expression placeholder option to your test